core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/CodeSystem.java

```diff
@@ -216,7 +216,7 @@ private Builder getSelf() {
 	private Map<String, Object> additionalProperties;
 	private List<CodeSystemURI> availableUpgrades;
 	private BranchInfo extensionOfBranchInfo;
-	private BranchInfo upgradeOfBranchInfo;
+	private UpgradeInfo upgradeOfBranchInfo;
```

Comment:
Rename field and getter/setter methods as well, not just the type. `upgradeOfInfo` or `upgradeInfo`.

Reply:
Done